drivers/pulse_counter/pulse_counter_saul.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_pulse_counter
```

Comment:
typo in group name: should be driver**s**_pulse_counter

Reply:
addressed